src/AccessibilityInsights.SharedUx/Controls/CustomControls/ScannerResultCustomListControl.xaml.cs

```diff
@@ -0,0 +1,266 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+using AccessibilityInsights.SharedUx.ViewModels;
+using AccessibilityInsights.Extensions.Helpers;
+using AccessibilityInsights.CommonUxComponents.Dialogs;
+using AccessibilityInsights.SharedUx.FileIssue;
+using System.IO;
+using AccessibilityInsights.Extensions.Interfaces.IssueReporting;
+using AccessibilityInsights.SharedUx.Telemetry;
+using AccessibilityInsights.SharedUx.Enums;
+
+namespace AccessibilityInsights.SharedUx.Controls.CustomControls
+{
+    /// <summary>
+    /// Interaction logic for ScannerResultCustomListControl.xaml
+    /// </summary>
+    public partial class ScannerResultCustomListControl : UserControl
+    {
+        private ScannerResultCustomListContext _controlContext;
+
+        public IEnumerable ItemsSource 
+        {
+            get => lvDetails.ItemsSource;
+            set { lvDetails.ItemsSource = value; } 
+        }
+
+        public int SelectedIndex { get; set; }
+
+        public ViewBase View { get; set; }
```

Comment:
This also feels disconnected. Maybe this?
```suggestion
        public ViewBase View => this.gvRules;
```